### PR TITLE
Update SVGs

### PR DESCRIPTION
**Pull request by hkamran80**

### CODE REVIEW

File: img/b/bugsnag.com.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" width="336.047" height="336.047"><g transform="translate(45.523 -4.473)" fill="#4949e4"><circle cx="122.5" cy="223.8" r="13.92"/><path d="M122.5 340.52A116.85 116.85 0 015.78 223.8v-57.88a8.69 8.69 0 018.68-8.69h41.4l-.14-134-32.58 20.05v76.3a8.68 8.68 0 01-17.36 0V42.29A15.67 15.67 0 0113.21 29L49.3 6.79a15.6 15.6 0 0123.78 13.29l.15 137.15h49.27a66.57 66.57 0 11-66.56 66.57l-.06-49.2H23.14v49.2a99.36 99.36 0 1099.36-99.36h-15.06a8.68 8.68 0 010-17.36h15.06a116.72 116.72 0 010 233.44zM73.25 174.6v49.19a49.2 49.2 0 1049.2-49.19z"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 233.3 336" fill="#4949e4"><g transform="translate(-5.8 -4.5)"><circle cx="122.5" cy="223.8" r="13.9"/><path d="M122.5 340.5A116.8 116.8 0 0 1 5.8 223.8v-57.9a8.7 8.7 0 0 1 8.7-8.7h41.4l-.2-134-32.6 20v76.4a8.7 8.7 0 0 1-17.3 0V42.3A15.7 15.7 0 0 1 13.2 29L49.3 6.8A15.6 15.6 0 0 1 73.1 20l.1 137.1h49.3a66.6 66.6 0 1 1-66.6 66.6v-49.2H23v49.2a99.4 99.4 0 1 0 99.4-99.4h-15a8.7 8.7 0 0 1 0-17.3h15a116.7 116.7 0 0 1 0 233.4zM73.2 174.6v49.2a49.2 49.2 0 1 0 49.3-49.2z"/></g></svg>
```
Comment (Carlgo11) [inline img/b/bugsnag.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#4949e4" viewBox="0 0 233.3 336"><circle cx="116.7" cy="219.3" r="13.9"/><path d="M116.7 336A116.8 116.8 0 0 1 0 219.3v-57.9a8.7 8.7 0 0 1 8.7-8.7h41.4l-.2-134-32.6 20v76.4a8.7 8.7 0 0 1-17.3 0V37.8c0-5.4 2.8-10.4 7.4-13.3L43.5 2.3a15.6 15.6 0 0 1 23.8 13.2l.1 137.1h49.3a66.6 66.6 0 1 1-66.6 66.6V170H17.2v49.2a99.4 99.4 0 1 0 99.4-99.4h-15a8.7 8.7 0 0 1 0-17.3h15a116.7 116.7 0 1 1 0 233.4ZM67.4 170.1v49.2a49.2 49.2 0 1 0 49.3-49.2Z"/></svg>
```

File: img/a/appfolio.com.svg [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" fill="#008fbe" viewBox="0 0 64 64"><path d="M42.4 30.8a274 274 0 0 1-9.1 1.6c-8.6 1.3-15.6 3-15.6 9.9 0 6 7 9.3 13 9.3C36 51.6 47 48.5 47 36.7v-8.1h-.2c-.4.7-2.3 1.9-4.3 2.2z"/><path d="M32 0a32 32 0 1 0 32 32A32 32 0 0 0 32 0zm19 44.1a19.2 19.2 0 0 1-11.5 11.6 24 24 0 0 1-9.7 1.4C22.4 57 12 52.2 12 44c0-4.1 1.2-12 17.5-14 11.8-1.4 14.2-1.4 16-2.8a5.8 5.8 0 0 0 1.8-4.6 9.8 9.8 0 0 0-2-6.5c-2.3-2.9-6.4-3.8-11.2-3.8-10.2 0-13 5-14.1 10.2a2.5 2.5 0 0 1-2.7 2.1 2.8 2.8 0 0 1-3-2.4C14.8 19 16.4 7 35 7c5.1 0 17 1.7 17 15.9 0 6.4.6 17.1-1 21.2z"/></svg>
```
Comment (Carlgo11) [inline img/a/appfolio.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#008fbe" viewBox="0 0 64 64"><path d="M32 0a32 32 0 1 0 32 32A32 32 0 0 0 32 0zm3 7c5.1 0 17 1.7 17 15.9 0 6.4.6 17.1-1 21.2a19.2 19.2 0 0 1-11.5 11.6 24 24 0 0 1-9.7 1.4C22.4 57 12 52.2 12 44c0-4.1 1.2-12 17.5-14 11.8-1.4 14.2-1.4 16-2.8a5.8 5.8 0 0 0 1.8-4.6 9.8 9.8 0 0 0-2-6.5c-2.3-2.9-6.4-3.8-11.2-3.8-10.2 0-13 5-14.1 10.2a2.5 2.5 0 0 1-2.7 2.1 2.8 2.8 0 0 1-3-2.4C14.8 19 16.4 7 35 7zm11.8 21.6c-.4.7-2.3 1.9-4.3 2.2h-.1a274 274 0 0 1-9.1 1.6c-8.6 1.3-15.6 3-15.6 9.9 0 6 7 9.3 13 9.3C36 51.6 47 48.5 47 36.7v-8.1h-.2z"/></svg>
```

File: img/2/23andme.com.svg [hunk 3]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20.57 39.8"><path fill="#92c746" d="M.34 29.21a3.16 3.16 0 0 0 1.47 4.28 3.3 3.3 0 0 0 4.38-1.43l1.32-2.58-2.72-8.95ZM18.77.34a3.3 3.3 0 0 0-4.38 1.42L9.06 12.18l2.72 8.95 8.45-16.51c.8-1.58.15-3.5-1.46-4.28"/><path fill="#d50f67" d="M9.95 37.52A3.28 3.28 0 0 0 14 39.67a3.18 3.18 0 0 0 2.2-3.97l-4.42-14.57-4.27 8.35 2.44 8.04ZM7.27 6.28a3.28 3.28 0 0 0-4.06-2.15 3.18 3.18 0 0 0-2.2 3.97l3.78 12.43 4.27-8.35z"/><path fill="#3ba510" d="m4.79 20.53 2.72 8.95 4.27-8.35-2.72-8.95z"/></svg>
```
Comment (Carlgo11) [inline img/2/23andme.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20.57 39.8"><path fill="#92c746" d="M14.4 1.76.33 29.21a3.16 3.16 0 0 0 1.47 4.28 3.3 3.3 0 0 0 4.38-1.43l1.32-2.58L20.23 4.62c.8-1.58.15-3.5-1.46-4.28a3.58 3.58 0 0 0-4.38 1.42z"/><path fill="#d50f67" d="M3.21 4.13a3.18 3.18 0 0 0-2.2 3.97l8.94 29.42A3.28 3.28 0 0 0 14 39.67a3.18 3.18 0 0 0 2.2-3.97L7.27 6.28c-.5-1.87-2.31-2.62-4.06-2.15Z"/><path fill="#3ba510" d="m4.79 20.53 2.72 8.95 4.27-8.35-2.72-8.95z"/></svg>
```

**Review by Carlgo11 — COMMENTED** [review]
Did some further cleanup :wink:

**Comment by 40504** [comment]
look at this file closer
Screenshot 2023-03-31 at 17 48 30

**Comment by hkamran80** [comment]
> look at this file closer Screenshot 2023-03-31 at 17 48 30

I'll fix that, thanks!

**Comment by hkamran80** [comment]
I'll reopen this when I have more time

**Comment by Carlgo11** [comment]
How about reopening the PR and merging the existing changes?
Further changes can be made in another PR later on.

**Comment by hkamran80** [comment]
Sure

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**